.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+lib:
+  - lib/**/*
+  - include/**/*
+
+core:
+  - db/**/*
+  - display/**/*
+  - general/**/*
+  - imagery/**/*
+  - misc/**/*
+  - ps/**/*
+  - raster/**/*
+  - raster3d/**/*
+  - scripts/**/*
+  - temporal/**/*
+  - vector/**/*
+
+GUI:
+  - gui/**/*
+
+Windows:
+  - mswindows/**/*
+
+MacOS:
+  - macosx/**/*
+
+Linux:
+  - rpm/**/*
+  - singularity/**/*
+
+docker:
```

Comment:
This label doesn't exist in GitHub yet, see also related discussion about labels at, https://trac.osgeo.org/grass/wiki/GitMigration#Migrationoftracissues

Reply:
`docker`is meanwhile there: https://github.com/OSGeo/grass/labels